### ISSUE TITLE
GitHub Action: Allow multiple files as input

### ISSUE THREAD

**Issue by stefanbuck** [RoadieHQ/backstage-entity-validator#15]
Hi, thank you so much for creating this validator. I've noticed that the input (in a Github Action context) can be only a single path. In our monorepo context this doesn't scale very well. Ideally I would like to be able to define a glob or pass in a list of files like

```
- uses: RoadieHQ/backstage-entity-validator@@0.3.0
  with:
    path: './a/service-a.yaml ./b/service-b.yaml ./c/service-c.yaml'
```

or as a glob

```
- uses: RoadieHQ/backstage-entity-validator@@0.3.0
  with:
    path: './*/service-*.yaml'
```

Would you accept such a contribution?

**Comment by Xantier** [comment]
Hi @stefanbuck ,

Thanks for noticing that! Yeah, definitely would be worthwhile addition to the repo. The CLI was update recently to accept multiple files as well with https://github.com/RoadieHQ/backstage-entity-validator/pull/13. Contribution would be very welcome to add this functionality

**Comment by Xantier** [comment]
Thanks again @stefanbuck. I've released v0.3.1 of GHA with these changes.
